python/ray/_private/runtime_env/packaging.py

```diff
@@ -132,19 +135,19 @@ def parse_uri(pkg_uri: str) -> Tuple[Protocol, str]:
                 netloc='_ray_pkg_029f88d5ecc55e1e4d64fc6e388fd103.zip'
             )
             -> ("gcs", "_ray_pkg_029f88d5ecc55e1e4d64fc6e388fd103.zip")
-    For HTTPS URIs, the netloc will have '.' replaced with '_', and
+    For HTTP(S) URIs, the netloc will have '.' replaced with '_', and
     the path will have '/' replaced with '_'. The package name will be the
-    adjusted path with 'https_' prepended.
+    adjusted path with 'http(s)_' prepended.
```

Comment:
Instead of `http(s)_`, could you say "`http_` or `https_`"? It's unclear whether the URI should have the string `http(s)_` prepended or either `http_` or `https_`.